doc/basics/compilation.md

```diff
@@ -152,7 +152,7 @@ Out-of source builds are mandatory, we recommend to follow the usual sequence, a
 
 ~~~{.bash}
 cmake . -B build-r -DCMAKE_BUILD_TYPE=Release -C external/install-r/radium-options.cmake -DQt5_DIR=path/to/qt5
-cmake --build build-r --target install --parallel
+cmake --build build-r --config Release --target install --parallel
```

Comment:
The lines above should be the same than in the TL;DR section. 
For building and installing, cmake recommend to use `cmake --build` then `cmake --install`.
the command `cmake --install` allows also to change the installation prefix but I found 2 problems with this as the files LICENCE and README.md will be still installed in the original directory.
For Qt, instead of setting directly a precise version, using `-DCMAKE_PREFIX_PATH=/path/to/qt/lib/cmake/` seems to me less restrictive